src/tests/ftest/daos_test/suite.py

```diff
@@ -302,6 +302,22 @@ def test_daos_rebuild_simple(self):
         """
         self.run_subtest()
 
+    def test_daos_rebuild_simple_interactive(self):
+        """Jira ID: DAOS-1568
```

Comment:
Nit - this gets copy pasted wrong all over but it should be tied to THIS PR. Or IMO you could leave it out since the git history has this info
```suggestion
        """Jira ID: DAOS-17354
```